taxi/forms.py

```diff
@@ -50,3 +50,36 @@ def validate_license_number(
         raise ValidationError("Last 5 characters should be digits")
```

Comment:
The error message mentions 'Last 5 characters should be digits', but the check is for the last 5 characters. Ensure that the logic and error message are consistent.